sashimi/config.py

```diff
@@ -20,6 +20,11 @@
     "scanning": "mock",
     "scopeless": False,
     "sample_rate": 40000,
+    "voxel_size" : {
+        "x": 0.3,
+        "y": 0.3,
+        "z": 6.5
```

Comment:
hmm, strange to have z voxel size here, it is determined by spacing between planes